### PR TITLE
fix: og-image working on www/usage/trpc

### PR DESCRIPTION
**Pull request by barrybtw**
Closes #<issue>

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

Added a condition to check if the astro props passed is an absolute url or a relative path and handle the og-image accordingly.
_www/src/components/headSeo.astro_ 
---

💯

### CODE REVIEW

File: .changeset/pretty-seahorses-cover.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"create-t3-app": patch
```
Comment (c-ehrlich) [inline .changeset/pretty-seahorses-cover.md:2]: I don't think we need changesets for PRs that don't change the CLI or outputted application

File: www/src/components/headSeo.astro [hunk 2]
```diff
@@ -15,7 +15,16 @@ const formattedContentTitle = frontmatter
   : SITE.title;
 
 const imageSrc = frontmatter?.image?.src ?? OPEN_GRAPH.image.src;
-const imageUrl = new URL(imageSrc, Astro.url);
+
+let imageUrl;
+
+// Check if image is a relative path or absolute URL, decide how to handle it accordingly
+if (imageSrc.startsWith("/")) {
```
Comment (c-ehrlich) [inline www/src/components/headSeo.astro:22]: I just merged the other PR for this, can you merge next to your branch and make sure it's all good?

File: www/src/layouts/docs.astro [hunk 3]
```diff
@@ -39,6 +39,37 @@ const githubEditUrl = `${CONFIG.GITHUB_EDIT_URL}/${currentFile}`;
     <title>
       {frontmatter.title} • {CONFIG.SITE.title}
     </title>
+    <script type="module" defer>
```
Comment (c-ehrlich) [inline www/src/layouts/docs.astro:42]: i think this is from your other pr?

**Comment by juliusmarminge** [comment]
Does this do anything that #637 didn't fix or should we close this?

**Comment by barrybtw** [comment]
Fix was made a minute before this PR.

**Pull request closed without merging**